metric/emitter/Godeps/Godeps.json

```diff
@@ -3,10 +3,19 @@
 	"GoVersion": "go1.8",
 	"GodepVersion": "v79",
 	"Packages": [
-		"github.com/influxdata/influxdb/client/v2"
+		"github.com/influxdata/influxdb/client/v2",
+		"github.com/prometheus/client_golang/prometheus"
```

Comment:
The aforementonied vendor issue. I couldn't get these deps into the `metric/emitter/vendor` dir reliably.